dplace_app/api_views.py

```diff
@@ -221,21 +220,29 @@ def id_array(l):
             result_set.languages.add(lang)
 
     if 'c' in query_dict:
+        f = open('text.txt', 'w')
+        f.write(query_dict['c'][0].split('-')[0])
```

Comment:
Ah! Sorry! Yes. Will remove now.


On 7/22/2016 6:55 PM, Robert Forkel wrote:
>
> In dplace_app/api_views.py 
> <https://github.com/D-PLACE/dplace/pull/393#discussion_r71834069>:
>
> > @@ -221,21 +220,29 @@ def id_array(l):
> >              result_set.languages.add(lang)
> >
> >      if 'c' in query_dict:
> > +        f = open('text.txt', 'w')
> > +        f.write(query_dict['c'][0].split('-')[0])
>
> Writing this text file was for debugging, right? So should be removed 
> before putting the code into production.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/D-PLACE/dplace/pull/393/files/51234edf31b1c3debd19003c92bc2a06aa509669#r71834069>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AIB9sOXJgPZverfRQ8bGnl50Zyk8ypbYks5qYGlmgaJpZM4JSY1s>.
>